src/react-hooks/buildHooks.ts

```diff
@@ -42,8 +42,49 @@ export type UseQuery<D extends QueryDefinition<any, any, any, any>> = <R = UseQu
   arg: QueryArgFrom<D>,
   options?: UseQuerySubscriptionOptions & UseQueryStateOptions<D, R>
 ) => UseQueryStateResult<D, R> & ReturnType<UseQuerySubscription<D>>;
+
 interface UseQuerySubscriptionOptions extends SubscriptionOptions {
+  /**
+   * Prevents a query from automatically running.
+   *
+   * @remarks
+   * When skip is true:
+   *
+   * - **If the query has cached data:**
+   *   * The cached data **will not be used** on the initial load, and will ignore updates from any identical query until the `skip` condition is removed
+   *   * The query will have a status of `uninitialized`
+   *   * If `skip: false` is set after skipping the initial load, the cached result will be used
+   * - **If the query does not have cached data:**
+   *   * The query will have a status of `uninitialized`
+   *   * The query will not exist in the state when viewed with the dev tools
+   *   * The query will not automatically fetch on mount
+   *   * The query will not automatically run when additional components with the same query are added that do run
```

Comment:
This reads as if it is still valid for `skip: false` and doesn't really seem right as is. Can we change that somehow?

Reply:
Sorry I'm not understanding exactly what you mean, can you elaborate a bit please?

Reply:
Okay, had to re-read it and it *does* make sense, it's just easy to get lost in the middle.

My reading path was
>    * When skip is true:
...
> If `skip: false` is set after skipping the initial load, the cached result will be used
...
> **If the query does not have cached data:**
...
>* The query will not automatically fetch on mount

By that point I had reset my brain with the mention of `skip: false` and everything after that seemed wrong. I believe that works better when not reading it as markdown in a github diff :rofl: